### DISCUSSION TITLE
nginx service is started on front end but website with public ip is not loading.

### DISCUSSION THREAD

**Discussion by sudhakatta** [learndevopsonline/learndevopsonline#129] [Q&A] [answered]
Hi @r-devops ,

I have enabled all the inbound rules on NSG,Still am not able to access the website via public ip.

Please see the attached screen.
[](u
<img width="933" alt="nsg rules" src="https://user-images.githubusercontent.com/125816794/220988304-a35272c8-e96a-4612-b54a-30ef367348a9.PN
<img width="933" alt="nsg rules" src="https://user-images.githubusercontent.com/125816794/220988401-70a950e2-e65a-4671-ac9b-27f79f0e5022.PNG">
G">
rl)

**Comment by akai257** [comment]
Can you go to the VM and share the screenshot of network adapter, it seems it has more than one NSG attached.

**Reply by sudhakatta** [reply]
> Can you go to the VM and share the screenshot of network adapter, it seems it has more than one NSG attached.
Hi Please find the below screens.
![image](https://user-images.githubusercontent.com/125816794/221170653-70ab91a7-b745-4701-946b-7f14855758b8.png)
![image](https://user-images.githubusercontent.com/125816794/221170825-e12c1bfe-9f03-4af7-bd77-d0dd57d29744.png)

**Comment by sudhakatta** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/125816794/221176349-0531b734-7fcc-4a5e-8f1e-06b130a78dfa.png)

Somehow it is working now, I have not made any changes today.
